scipy/optimize/tests/test_optimize.py

```diff
@@ -223,6 +223,27 @@ def test_powell(self):
         assert_(self.funccalls <= 116 + 20, self.funccalls)
         assert_(self.gradcalls == 0, self.gradcalls)
 
+    @pytest.mark.xfail
```

Comment:
Could be a bit weird, but an option is to use `strict=True`. This way CI will fail again when the test will pass. This would force us to have a look at this again.

Appart from that, I like to have `reason="..."` so that we don't need to go check the file for eventual comments explaining why.

Reply:
I would have used `strict` but it doesn't fail on all platforms.
I'll add the r`eason`. (I tried, actually, but it wasn't the first positional argument and i didn't immediately find the name of the keyword argument.)

Reply:
Oh interesting, thanks I didn't know this wasn't working everywhere.

Reply:
Added `reason`.